layer/layer/canary_stack.py

```diff
@@ -16,29 +20,74 @@ def __init__(
         construct_id: str,
         powertools_version: str,
         ssm_paramter_layer_arn: str,
+        ssm_parameter_layer_arm64_arn: str,
         **kwargs,
     ) -> None:
         super().__init__(scope, construct_id, **kwargs)
 
-        VERSION_TRACKING_EVENT_BUS_ARN: str = (
-            "arn:aws:events:eu-central-1:027876851704:event-bus/VersionTrackingEventBus"
-        )
+        deploy_stage = CfnParameter(
+            self, "DeployStage", description="Deployment stage for canary"
+        ).value_as_string
 
         layer_arn = StringParameter.from_string_parameter_attributes(
             self, "LayerVersionArnParam", parameter_name=ssm_paramter_layer_arn
         ).string_value
+        Canary(
+            self,
+            "Canary-x86-64",
+            layer_arn=layer_arn,
+            powertools_version=powertools_version,
+            architecture=Architecture.X86_64,
+            stage=deploy_stage,
+        )
+
+        layer_arm64_arn = StringParameter.from_string_parameter_attributes(
+            self,
+            "LayerArm64VersionArnParam",
+            parameter_name=ssm_parameter_layer_arm64_arn,
+        ).string_value
+        Canary(
+            self,
+            "Canary-arm64",
+            layer_arn=layer_arm64_arn,
+            powertools_version=powertools_version,
+            architecture=Architecture.ARM_64,
+            stage=deploy_stage,
+        )
+
+
+class Canary(Construct):
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        layer_arn: str,
+        powertools_version: str,
+        architecture: Architecture,
+        stage: str,
+    ):
+        super().__init__(scope, construct_id)
 
-        layer = LayerVersion.from_layer_version_arn(self, "PowertoolsLayer", layer_version_arn=layer_arn)
-        deploy_stage = CfnParameter(self, "DeployStage", description="Deployment stage for canary").value_as_string
+        layer = LayerVersion.from_layer_version_arn(
+            self, "PowertoolsLayer", layer_version_arn=layer_arn
+        )
 
-        execution_role = Role(self, "LambdaExecutionRole", assumed_by=ServicePrincipal("lambda.amazonaws.com"))
+        execution_role = Role(
+            self,
+            "LambdaExecutionRole",
+            assumed_by=ServicePrincipal("lambda.amazonaws.com"),
+        )
 
         execution_role.add_managed_policy(
-            ManagedPolicy.from_aws_managed_policy_name("service-role/AWSLambdaBasicExecutionRole")
+            ManagedPolicy.from_aws_managed_policy_name(
+                "service-role/AWSLambdaBasicExecutionRole"
+            )
         )
 
         execution_role.add_to_policy(
-            PolicyStatement(effect=Effect.ALLOW, actions=["lambda:GetFunction"], resources=["*"])
+            PolicyStatement(
+                effect=Effect.ALLOW, actions=["lambda:GetFunction"], resources=["*"]
```

Comment:
@am29d please remind me again why do we need this permission in the Lambda itself? leftover?

Looking at the new canary I don't see an API call that would require this permission.